### PR TITLE
Added Tempo; Added instrumentation

### PR DESCRIPTION
**Pull request by bwplotka**

### CODE REVIEW

File: source/tracing.mimic.go [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+						BlockRetention:          1 * time.Hour,
+						CompactedBlockRetention: 10 * time.Minute,
+					},
+				},
+				StorageConfig: storage.Config{
+					Trace: tempodb.Config{
+						Backend: "local",
+						Block: &encoding.BlockConfig{
+							BloomFP:              .05,
+							IndexDownsampleBytes: 1000,
```
Comment (joe-elliott) [inline source/tracing.mimic.go:81]: This is a very small value. We use 1MB with zstd.

Comment (joe-elliott) [inline source/tracing.mimic.go:81]: This is rather low. We use 1MB with zstd encoding. It controls the total bytes that a single index record references in the backend.

File: source/tracing.mimic.go [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+							"protocols": map[string]interface{}{
+								"grpc": nil,
+							},
+						},
+					},
+				},
+				Compactor: compactor.Config{
+					Compactor: tempodb.CompactorConfig{
+						MaxCompactionRange:      1 * time.Hour,
+						MaxBlockBytes:           1e6,
```
Comment (joe-elliott) [inline source/tracing.mimic.go:71]: This is not necessarily wrong, but also low. Ours is set to 10GB.

**Review by joe-elliott — COMMENTED** [review]
Unsure what the scaling expectations are of Tempo being deployed in this manner. Made a few comments that might help improve understanding of what the options do!

**Comment by bwplotka** [comment]
This is super tiny demo setup that has to run (ideally) in Katacoda =D So 🤞🏽 Thanks for tips!

**Comment by bwplotka** [comment]
Added instrumentation cc @AnaisUrlichs 🤗

**Comment by bwplotka** [comment]
image

**Comment by bwplotka** [comment]
To scrape exemplars, Prometheus has to set headers `Accept` `application/openmetrics-text; version=0.0.1; charset=utf-8`

image

**Review by AnaisUrlichs — APPROVED** [review]

**Pull request merged**